src/wp-includes/rest-api/endpoints/class-wp-rest-templates-controller.php

```diff
@@ -865,6 +865,7 @@ private static function get_wp_templates_author_text_field( $template_object ) {
 			case 'site':
 				return get_bloginfo( 'name' );
 			case 'user':
+			default:
```

Comment:
I like the idea of making this code future-proof. But I'm thinking a better approach is to decouple the 2 methods.

How so?

Right now, `get_wp_templates_original_source_field()` returns one of the specific `case`s that are already covered here. What happens if that method gets changed, but changes are not reflected in this method?

I suggest instead of making the default use the `user`, instead, return an empty string at the end of the method. This `return` provides a graceful fallback or fallthrough.

```
return '';
```

What do you think @anton-vlasenko ?

Reply:
Well spotted, @hellofromtonya!
Thank you for the review.
Initially, I had the same idea but can't recall why I chose the other option.
Fixed in 96b5d550f1a64c3ddc0338b51e4036b580c59f7f.